scripts/main.js

```diff
@@ -1,127 +1,121 @@
+  const cardPages = document.createElement('p');
+  cardPages.classList.add(...bookTextClasses, 'fs-5');
+  const cardPagesText = document.createTextNode(`Num. of pages: ${book.numPages}`);
   cardPages.appendChild(cardPagesText);
   cardBody.appendChild(cardPages);
 
-  let btnsContainer = document.createElement("div");
-  btnsContainer.classList.add("d-grid", "gap-2", "col-6", "mx-auto")
+  const btnsContainer = document.createElement('div');
+  btnsContainer.classList.add('d-grid', 'gap-2', 'col-6', 'mx-auto');
   cardBody.appendChild(btnsContainer);
 
-  let readBtn = document.createElement("button");
+  const readBtn = document.createElement('button');
   let cardReadText;
-  if (book.read === true){
+  if (book.read === true) {
     readBtn.classList.add(...readClasses);
-    cardReadText = document.createTextNode("Read: Yes");
-  }else{
+    cardReadText = document.createTextNode('Read: Yes');
+  } else {
     readBtn.classList.add(...notReadClasses);
-    cardReadText = document.createTextNode("Not Read Yet");
+    cardReadText = document.createTextNode('Not Read Yet');
   }
 
-  readBtn.setAttribute("id", "read" + book.title)
-  readBtn.addEventListener("click", changeStatus)
+  readBtn.setAttribute('id', `read${book.title}`);
+  readBtn.addEventListener('click', changeStatus);
   readBtn.appendChild(cardReadText);
   btnsContainer.appendChild(readBtn);
-  
 
-  let deleteBtn = document.createElement("button");
-  deleteBtn.setAttribute("id", "delete" + book.title)
-  deleteBtn.setAttribute('data-index', myLibrary.indexOf(book))
-  deleteBtn.addEventListener("click", (e) =>{
+
+  const deleteBtn = document.createElement('button');
+  deleteBtn.setAttribute('id', `delete${book.title}`);
+  deleteBtn.setAttribute('data-index', myLibrary.indexOf(book));
+  deleteBtn.addEventListener('click', (e) => {
     myLibrary.splice(e.target.dataset.index, 1);
     card.remove();
     saveToLocalStorage();
-  }) 
-  let deleteBtnText = document.createTextNode("Delete book");
-  deleteBtn.classList.add("btn", "btn-danger")
+  });
+  const deleteBtnText = document.createTextNode('Delete book');
+  deleteBtn.classList.add('btn', 'btn-danger');
   deleteBtn.appendChild(deleteBtnText);
   btnsContainer.appendChild(deleteBtn);
 }
 
-function changeStatus(book){
-  if ("Read: Yes" === true) {
-   book.read = false
- }
- else{
-   book.read = true
- } 
-}
+function addBookToLibrary(e) {
+  e.preventDefault();
 
-function retrieveLocalStorage() {
-  // if (localStorage.getItem("mylibrary")) {
-    myLibrary = JSON.parse(localStorage.getItem("library"));
-  // } else {
-  //   library = DEFAULT_DATA;
-  // }
+  const newBook = new Book(title.value, author.value, numPages.value, read.checked);// eslint-disable-line
+  if (title.value.length === 0 || author.value.length === 0 || numPages.value === '') {// eslint-disable-line
+    alert('Please, fill all the fields');// eslint-disable-line
```

Comment:
- [ ] Do not use Window.alert() or Window.confirm(). It’s ugly, impossible to style, stop code execution and display differently on different browsers. Use custom modal instead. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/alert_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/javascript-module-v-10/166782/path/step/97634928/))